### DISCUSSION TITLE
Single phase UK use

### DISCUSSION THREAD

**Discussion by bry25uk** [emporia-vue-local/esphome#156] [General]
Hi,
Just flashed my VUE 2 with this and connected seemlessly, however some sensors appear not to be reporting. I was wondering if in this section :

Pay close attention to set the phase_id for each breaker by matching it to the phase/leg it connects to in the panel
  - { phase_id: phase_a, input:  "1", power: { name:  "Circuit 1 Power", id:  cir1, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_b, input:  "2", power: { name:  "Circuit 2 Power", id:  cir2, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_a, input:  "3", power: { name:  "Circuit 3 Power", id:  cir3, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_a, input:  "4", power: { name:  "Circuit 4 Power", id:  cir4, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_a, input:  "5", power: { name:  "Circuit 5 Power", id:  cir5, filters: [ *moving_avg, *pos, multiply: 2 ] } }
  - { phase_id: phase_a, input:  "6", power: { name:  "Circuit 6 Power", id:  cir6, filters: [ *moving_avg, *pos, multiply: 2 ] } }
  - { phase_id: phase_a, input:  "7", power: { name:  "Circuit 7 Power", id:  cir7, filters: [ *moving_avg, *pos, multiply: 2 ] } }
  - { phase_id: phase_b, input:  "8", power: { name:  "Circuit 8 Power", id:  cir8, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_b, input:  "9", power: { name:  "Circuit 9 Power", id:  cir9, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_b, input: "10", power: { name: "Circuit 10 Power", id: cir10, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_a, input: "11", power: { name: "Circuit 11 Power", id: cir11, filters: [ *moving_avg, *pos, multiply: 2 ] } }
  - { phase_id: phase_a, input: "12", power: { name: "Circuit 12 Power", id: cir12, filters: [ *moving_avg, *pos, multiply: 2 ] } }
  - { phase_id: phase_a, input: "13", power: { name: "Circuit 13 Power", id: cir13, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_a, input: "14", power: { name: "Circuit 14 Power", id: cir14, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_b, input: "15", power: { name: "Circuit 15 Power", id: cir15, filters: [ *moving_avg, *pos ] } }
  - { phase_id: phase_a, input: "16", power: { name: "Circuit 16 Power", id: cir16, filters: [ *moving_avg, *pos ] } }
  
  DO I have to change them all to phase A as I only have single phase?
  
  Thanks
  Bry25

**Comment by flaviut** [comment]
> DO I have to change them all to phase A as I only have single phase?

Yes

**Comment by Infernoken76** [comment]
Question I have is why you would need the multiply: 2 for some 
I have a 3phase network 3L+N, and al my circuits are single phase…. (L+N)

So don’t understand the need for the multiply: 2 in such cases, some removed it in my yaml file.
Same for the above question for a single phase UK residence…

Or is there another reason?

**Reply by flaviut** [reply]
If all your circuits are L+N, then you wouldn't need the `multiply: 2` and can just remove it. For example,

```
{ phase_id: phase_a, input: "5", power: { name: "Circuit 5 Power", id: cir5, filters: [ *moving_avg, *pos, multiply: 2 ] } }
```

becomes

```
{ phase_id: phase_a, input: "5", power: { name: "Circuit 5 Power", id: cir5, filters: [ *moving_avg, *pos ] } }
```

**Reply by Infernoken76** [reply]
> If all your circuits are L+N, then you wouldn't need the `multiply: 2` and can just remove it. For example,
> 
> ```
> { phase_id: phase_a, input: "5", power: { name: "Circuit 5 Power", id: cir5, filters: [ *moving_avg, *pos, multiply: 2 ] } }
> ```
> 
> becomes
> 
> ```
> { phase_id: phase_a, input: "5", power: { name: "Circuit 5 Power", id: cir5, filters: [ *moving_avg, *pos ] } }
> ```

Thanks, that is what I thought...

@bry25uk, I assume your UK single phase is also L+N, so you would also not need the multiply: 2
